sdk/identity/azure-identity/README.md

```diff
@@ -297,6 +297,31 @@ variables:
 
 Configuration is attempted in the preceding order. For example, if values for a client secret and certificate are both present, the client secret is used.
 
+### Configuring DefaultAzureCredential
```

Comment:
In the interest of slimming down the README, let's remove this section. I'll document the missing details over at https://learn.microsoft.com/azure/developer/python/sdk/authentication/credential-chains?tabs=dac#how-to-customize-defaultazurecredential.